homeassistant/components/deconz/light.py

```diff
@@ -47,7 +49,7 @@ def async_add_light(lights):
 
         for light in lights:
             if (
-                light.type not in COVER_TYPES + LOCK_TYPES + SWITCH_TYPES
+                light.type not in CONTROLLER + COVER_TYPES + LOCK_TYPES + SWITCH_TYPES
```

Comment:
This full list is calculated for each light. Better rewrite it to be 

`light.type not in CONTROLLER and light.type not in COVER_TYPES`

Or `all(light.type in collection for collection in [CONTROLLER, COVER_TYPES, ...])`